##########
## AUBO ##
##########

tool_dynamics:
  # The kinematics parameters of the tool, if tool is loaded on the end-effector,
  # the parameter should be set according to the actual needs;
  # if tool is not loaded, each section of this parameter should be set to 0.
  position: { x: 0.0, y: 0.0, z: 0.0 }
  # position: { x: 0.000, y: 0.000, z: 0.072 }
    # Tool position
  payload: 0.0
  # payload: 1.0
    # The payload of the tool [Kg]
  inertia: { xx: 0.0, xy: 0.0, xz: 0.0, yy: 0.0, yz: 0.0, zz: 0.0 }
  # inertia: { xx: 0.001293, xy: 0.0, xz: 0.0, yy: 0.002438, yz: 0.0, zz: 0.0003398 }
    # Tool's inertial matrix

collision_class: 8
  # The collision class of the manipulator, in the range [1-10].

max_joint_velocity: [ 2.62, 2.62, 2.62, 3.14, 3.14, 3.14 ]
  # The maximum joint velocity, the value si no more than [rad/s]
max_joint_acceleration: [ 2.62, 2.62, 2.62, 3.14, 3.14, 3.14 ]
  # The maximum joint acceleration, the value is no more than 3.14 [rad/s^2]

max_linear_velocity: 5.0
  # The maximum linear velocity of the end-effector, the value is no more than 5.0 [m/s]
max_linear_acceleration: 5.0
  # The maximum linear acceleration of the end-effector, the value is no more than 5.0 [m/s^2]
max_angular_velocity: 3.14
  # The maximum angular velocity of the end-effector, the value is no more than 3.14 [rad/s]
max_angular_acceleration: 3.14
  # The maximum angular acceleration of the end-effector, the value is no mora than 3.14 [rad/s^2]

blend_radius: 0.03
  #
